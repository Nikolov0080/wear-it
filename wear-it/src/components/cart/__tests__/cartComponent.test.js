import React from 'react';
import renderer from 'react-test-renderer';
import CartComponent from '../CartComponent';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Cart Component', () => {
    const tree = renderer
        .create(<BrowserRouter><CartComponent /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});