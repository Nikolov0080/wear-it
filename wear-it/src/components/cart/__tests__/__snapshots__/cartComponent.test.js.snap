// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Cart Component 1`] = `
<div>
  <div>
    <div
      className="loading"
    >
      Loading....
    </div>
  </div>
</div>
`;
